youtube_dl/extractor/rai.py

```diff
@@ -240,20 +222,32 @@ class RaiPlayLiveIE(RaiBaseIE):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, display_id)
+        media = self._download_json(
+            '%s.json' % urljoin(RaiBaseIE._BASE_URL, 'dirette/' + display_id),
```

Comment:
capture the base URL in `_VALID_URL` regular expression and then add `.json` suffix(same for `RaiPlayPlaylistIE`).

Reply:
This should be addressed by https://github.com/ytdl-org/youtube-dl/pull/27077/commits/191d41a86a3ff846f132c7e2b97f4db6525bacb4